hooks/pre/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,57 @@
+scenario = YAML.load_file('./config/last_scenario.yaml').fetch(:answer_file)
```

Comment:
This needs to use a full path to resolve. I think you can use something like `File.join(kafo.class.scenario_manager.config_dir, SCENARIO_NAME)` potentially. Usually though we use a module enabled check. That is, checking to see if 'katello' is enabled for example.